modules/ezcontent_block/modules/ezcontent_block_media/ezcontent_block_media.info.yml

```diff
@@ -0,0 +1,8 @@
+name: Media Block
+type: module
+description: "Provides Media Block Types Based Features."
+package: EZContent Block
+core: 8.x
```

Comment:
Let's remove this key as well, as we are not supporting the Drupal versions prior to 8.8.